Utils/patchArticles.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+
+const NCnewsAPI = axios.create({
+  baseURL: "https://nc-news-tm65.onrender.com",
+});
+
+export const patchArticle = (id, obj) => {
+return NCnewsAPI.patch(`/api/articles/${id}`, obj)
+      .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
Shouldn't really have a catch here in your api file. If you need a catch chain it when you invoke the function in your components.